functions/v2/firebase/hello-firestore/main.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_cloudevent_firebase_firestore]
+import functions_framework
+from google.events.cloud import firestore
+
+
+@functions_framework.cloud_event
+def hello_firestore(cloud_event):
+    """Triggered by a change to a Firestore document.
```

Comment:
nit: method docstrings should be descriptive-style (`triggers`).
```suggestion
    """Triggers by a change to a Firestore document.
```